nearswap/src/internal.rs

```diff
@@ -129,48 +118,75 @@ impl NearSwap {
         ynear_out: Balance,
     ) {
         let user = env::predecessor_account_id();
+
+        let in_bal = p.tokens;
+        let out_bal = p.ynear;
+        let in_amount = token_in;
+
+        let liquidityFee = self.calcLiquidityFee(in_bal, in_amount, out_bal);
+        let amount_out = self.calc_out_amount(in_amount, in_bal, out_bal);
+        
+        assert!(amount_out >= ynear_out, "ERR_MIN_AMOUNT");
```

Comment:
let's use same update as above.
Also, let's move and define `"ERR_MIN_AMOUNT"` in errors.rs

Reply:
ok